index.html

```diff
@@ -18,13 +18,18 @@
             <header>
                 <nav class="navbar">
                     <a class="logo" href="">Hamdullah</a>
-                    <img src="images/Union.png" alt="">
+                    <img class="logo-image" src="images/Union.png" alt="">
+                    <div class="menu-bar">
+                        <a class="portfolio-menu" href="">Portfolio</a>
+                        <a class="about-menu" href="">About</a>
+                        <a class="contact-menu" href="">Contact</a>
+                    </div>
                 </nav>
             </header>
 
             
            <!-- The main section  -->
-            <main>
+            <main class="main-class">
```

Comment:
- [ ] Kindly remove the empty space on the right side of the website 👍 
![Animation](https://user-images.githubusercontent.com/50721479/194669038-47988fbe-0898-407f-a512-e5c6cb13ab66.gif)